src/200-WIDGETS/storeInventory.twee

```diff
@@ -5,13 +5,27 @@
 <<set _ST = _SE.OpenStore(setup.player, _NPC);>>\
 <<set _CI = _ST.GetCommonInventory();>>\
 <<set _RI = _ST.GetRareInventory(); >>\
+<<set _DUR = _ST.DaysUntilRestocking() >>\
+<<if _DUR > 0>>\
+	<<set _restockingText to "Restocking in " + _DUR + " days">>\
+<<else>>\
+	<<set _restockingText to "Fresh supplies arrived this morning!">>\
+<</if>>\
+<<if SugarCube.settings.inlineDetailsInShop>>\
```

Comment:
When I tried to put <<if>> inside of the <td>, it escaped quotes and broke the table. Hence two variables are always set and used below unconditionally. I would appreciate a help to simplify this.